### PR TITLE
Update technische-universitat-munchen-controlling.csl

### PR DESCRIPTION
**Pull request by zuphilip**
The hwr-berlin style was renamed, cf.
https://github.com/citation-style-language/styles/blob/e09bfc04fe0dd4dd3090bb06125eca573f4da484/renamed-styles.json#L289

### CODE REVIEW

**Comment by zuphilip** [comment]
This search shows also that these are all occurences: https://github.com/citation-style-language/styles/search?utf8=%E2%9C%93&q=hwr&type=

**Comment by adam3smith** [comment]
thanks!

File: technische-universitat-munchen-controlling.csl [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <id>http://www.zotero.org/styles/technische-universitat-munchen-controlling</id>
     <link href="http://www.zotero.org/styles/technische-universitat-munchen-controlling" rel="self"/>
     <link href="http://www.zotero.org/styles/universitat-freiburg-geschichte" rel="template"/>
-    <link href="http://www.zotero.org/styles/hwr-berlin" rel="template"/>
+    <link href="http://www.zotero.org/styles/hochschule-fur-wirtschaft-und-recht-berlin" rel="template"/>
```
Comment (rmzelle) [inline technische-universitat-munchen-controlling.csl]: @adam3smith, @zuphilip, only now seeing this, but I'm confused.

This template style was renamed in 2013 (see https://github.com/citation-style-language/styles/commit/d6754738d27fd528775648714ce36ef269373568#diff-2657503fcaf93aaf9836fa01c692099eR22 and https://github.com/citation-style-language/styles/commit/d6754738d27fd528775648714ce36ef269373568#diff-2657503fcaf93aaf9836fa01c692099eR22). Shouldn't our automatic tests have caught this way back? I'm pretty sure we're testing whether the template style exists in the current repository.

Comment (adam3smith) [inline technische-universitat-munchen-controlling.csl]: We definitely do, yes -- without actually having looked at the code, I'm guessing the test might just look if the first template exists. This style has two.

Comment (rmzelle) [inline technische-universitat-munchen-controlling.csl]: Right, that's it. Just discovered that independently :P.

(the code for checking the template link is at https://github.com/citation-style-language/styles/blob/a6646ba5657dd22f3f4a534f6190b4a45be2895b/spec/styles_spec.rb#L26)

Comment (rmzelle) [inline technische-universitat-munchen-controlling.csl]: Okay, (low-priority) issue created: https://github.com/citation-style-language/styles/issues/2802

**Pull request merged**